elmslie-core/src/main/kotlin/money/vivid/elmslie/core/store/Actor.kt

```diff
@@ -17,4 +26,21 @@ abstract class Actor<Command : Any, Event : Any> {
         errorMapper: (error: Throwable) -> Event? = { null },
     ) = mapNotNull { eventMapper(it) }
         .catch { errorMapper(it)?.let { event -> emit(event) } ?: throw it }
+
+    protected fun <T : Any, Command : Any> Flow<T>.asSwitchFlow(command: Command, delayMillis: Long = 0): Flow<T> {
```

Comment:
it will be helpful to have a method with a custom key (the same command can run multiply switchers)

something like this:
```
protected fun <T : Any, Command : Any> Flow<T>.asSwitchFlow(key: Any, delayMillis: Long = 0) =

protected fun <T : Any, Command : Any> Flow<T>.asSwitchFlow(command: Command, delayMillis: Long = 0) =
    asSwitchFlow(command::kclass, delayMillis)
```

Reply:
the same command can run multiply switchers - it's not valid usage this thing. `switcher` is ment to be applied to final flow, not to sub-flows

Reply:
so I don't want to make possible run multiple switchers on the same command)